R/build-logo.R

```diff
@@ -1,29 +1,98 @@
 build_logo <- function(pkg = ".") {
+
   pkg <- as_pkgdown(pkg)
 
-  logo_path <- find_logo(pkg$src_path)
-  if (is.null(logo_path))
+  if (!dir_exists(path(pkg$src_path, "pkgdown", "favicon"))) {
     return()
+  }
+
+  dir_copy_to(pkg, path(pkg$src_path, "pkgdown", "favicon"), pkg$dst_path)
```

Comment:
Message should probably go here now

Reply:
Hum, what do you mean? 

`dir_copy_to` already prints an explicit message about what is happening

![screenshot from 2018-11-13 15-14-02](https://user-images.githubusercontent.com/10783929/48418917-dca7fa80-e756-11e8-85b8-22d95938ad08.png)

Reply:
Oh I forgot about that!